Pod/Extensions/Forgot Password/SKYAuthContainer+ForgotPassword.h

```diff
@@ -28,7 +28,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (void)resetPasswordWithUserID:(NSString *)userID
                            code:(NSString *)code
-                       expireAt:(NSDate *)expireAt
+                       expireAt:(long)expireAt
```

Comment:
Would we use `NSTimeInterval` for timestamp?

Reply:
I don't have strong preference in this case. However, [`NSTimeInterval`](https://developer.apple.com/documentation/foundation/nstimeinterval) is just a double, our API accepts integer, right? Having `NSTimeInterval` does not help much in this case. What do you think?

Reply:
Just discussed with kuen kuen, he agree to use `long` in this case. Please ignore my previous comment.  

Would like to have your help to check the travis build then. Thanks howa!